src/main/java/com/eprosima/fastdds/idl/templates/SerializationSource.stg

```diff
@@ -21,12 +21,15 @@ $fileHeader(ctx=ctx,  file=[ctx.filename, "SerializationSource.cpp"], descriptio
 
 #include "$ctx.filename$Serialization.h"
 
+#include <cassert>
+#include <inttypes.h>
+#include <stdint.h>
```

Comment:
Instead of including the C header files, I suggest including the C++ ones:

```suggestion
#include <cinttypes>
#include <cstdint>
```

Reply:
Done in 020947b